pillar/spoonbill.sls

```diff
@@ -0,0 +1,22 @@
+docker:
+  user: deployer
+  uid: 1002
+  docker_compose:
+    version: 1.29.2
+
+docker_apps:
+  spoonbill:
+    target: spoonbill
+    env:
+      DOMAIN: &DOMAIN flatten.open-contracting.org
+      API_DOMAIN: *DOMAIN
+      TRAEFIK_IP: 65.21.93.141
+      ACME_EMAIL: shakh@quintagroup.org
+      DEBUG: 1
```

Comment:
Really?

Reply:
Sorry, I used the same variables and values as the current deployment and forgot to remove this one.